tests/toolchain/gcc_fortran_compilation.pm

```diff
@@ -30,7 +30,10 @@ sub run {
     script_run 'cp FM923.DAT fcvs21_f95/';
     script_run 'pushd fcvs21_f95';
     # gfortran (and gcc) fixed to version in SLE12 after the yearly gcc update with Toolchain module
-    my $fortran_version = is_sle && sle_version_at_least('15') ? "gfortran" : "gfortran-5";
+    my $fortran_version = "gfortran-5";
+    if ((is_sle && sle_version_at_least('15')) or (is_leap and leap_version_at_least('15.0') or is_tumbleweed)) {
+        $fortran_version = "gfortran";
+    }
```

Comment:
hmm, if you need to ask for tumbleweed, you probably did it wrong ;)

What about...?

```perl
my $fortran_version = "gfortran";
if (!sle_version_at_least('15') and leap_version_at_least('15.0')) {
  $fortran_version = "gfortran-5";
}
```

Reply:
if it is caasp or other thing sle_version_at_least would die. I could empty the else condition, but not sure if it would affect other products, caasp for sure, as another college told me.